locales/cs.json

```diff
@@ -1,6 +1,7 @@
 {
     "nav.run": "Spustit",
     "nav.run_tt": "spustit tento databázový dotaz v rozhraní Overpass API",
+    "nav.clear": "Clear query",
```

Comment:
I am pretty sure that only the main language file should be changed.
The other languages will get the translations later.